xmtp_mls/src/groups/mod.rs

```diff
@@ -15,8 +14,6 @@ use openmls::{
 };
 use openmls_traits::OpenMlsProvider;
 use std::mem::discriminant;
-#[cfg(test)]
-use std::println as debug;
```

Comment:
I removed this; I assume it was here because RUST_LOG=debug spits out a bunch of irrelevant stuff. Can always use `RUST_LOG=xmtp_mls=debug` to filter out any logs not part of the `xmtp_mls` crate. per-module filtering also works, `RUST_LOG=xmtp_mls::groups=debug`

Reply:
@richardhuaaa put this in, but I believe the intention was just to get the logs to emit when running unit tests

Reply:
ah, i've been adding `crate::tests::setup` to unit tests if I need logs, should I go through and add that to each test you think? I've just been adding it as I go. Can also revert this if this way is easier

Reply:
In my perfect world, all unit tests would emit logs from our application automatically. And we could optionally set a command line flag to also get unit tests from libraries we use. Being able to set something per-file is second-best, and per-test is third.

Reply:
That's fair -- I've added `ctor` to setup this logging before any tests run, it works without any manual calls now

Reply:
@insipx, thank you for doing this - I couldn't figure out how to push all logs to print when tests fail. Does the `tracing_subscriber` automatically add all `log::` calls to the trace?